nick-questions.txt

```diff
@@ -0,0 +1,25 @@
+
+1. Why is the memo field in report so small?  Sourabh mentioned that this was so the whole report could be transmitted via bluetooth.  But in order for contact tracing to be effective, the client needs to download potentially thousands of these reports, i.e. at least one per infected user in the area, to see if they've been exposed.  How can they download these via Bluetooth?  If we think this is an impractical method to download thousands, and instead rely on http, then there's no reason to keep memo small.  Memo can be a rich source of data and I think it should be large enough to answer as-yet-unknown questions that epidemiologists may want to ask.

```

Comment:
The memo field is small because reports are downloaded by all clients. 255 bytes is still fairly large, and it's big enough to include a memo ID that references a longer body, if someone really needs that data.